core/deepmind.py

```
@@ -379,6 +379,24 @@ async def organize_and_inject_memories(
+                if memory_runtime and hasattr(memory_runtime, 'recall_by_sender_tag'):
+                    cross_memories = await memory_runtime.recall_by_sender_tag(
+                        f"uid:{sender_id}", limit=10
+                    )
+                    if cross_memories:
+                        existing_ids = {getattr(m, 'id', '') for m in long_term_memories}
+                        for mem in cross_memories:
+                            if getattr(mem, 'id', '') not in existing_ids:
+                                existing_ids.add(mem.id)
+                                long_term_memories.append(mem)
+            except Exception:
+                pass
```

Ruff (0.15.13)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)